commcare_connect/opportunity/api/serializers.py

```diff
@@ -38,3 +38,11 @@ class Meta:
 class UserLearnProgressSerializer(serializers.Serializer):
     completed_modules = serializers.IntegerField()
     total_modules = serializers.IntegerField()
+
+
+class UserVisitSerializer(serializers.ModelSerializer):
+    deliver_form_name = serializers.CharField(source="deliver_form.name")
+
+    class Meta:
+        model = UserVisit
+        fields = ["id", "status", "visit_date", "deliver_form_name"]
```

Comment:
should this also include  `xmlns` so forms can be matched if the name changes, and because its a bit nicer for machines than `deliver_form_name`.

Maybe @snopoke has thoughts?

Reply:
Added. https://github.com/dimagi/commcare-connect/pull/78/commits/c1d56f49361c789fb99e7dd0c5b33bb487aa14b8